composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
+        "php": "^5.6 || ^7.0",
         "psr/http-message": "^1.0"
```

Comment:
Why does this package depend on `http-message`? Those interfaces are not used here.

Reply:
https://github.com/php-fig/http-message-util/issues/13